e2e/home.component.e2e-spec.ts

```diff
@@ -11,9 +11,11 @@ describe('Home Component', () => {
 
     // Host URL + SPA Name automatically included
     SkyHostBrowser.get('/');
-    expect(element(by.tagName('h1')).getText()).toBe('Stache 2');
 
-    // // Since we've said this is an async test, we need to call done
-    done();
+    element(by.tagName('h1')).getText()
+      .then((text: string) => {
+        expect(text).toBe('Stache 2');
+        done();
```

Comment:
Yep, but the idea is that they need to update the tests to support their site, or remove them altogether. The vanilla template expects the same things from their users: https://github.com/blackbaud/skyux-template/blob/master/e2e/home.component.e2e-spec.ts#L15